mlir/include/mlir/Dialect/SparseTensor/IR/SparseTensorAttrDefs.td

```diff
@@ -162,6 +162,8 @@ def SparseTensorEncodingAttr : SparseTensor_Attr<"SparseTensorEncoding",
 
     - **nonunique** : duplicate coordinates may appear at the level
     - **nonordered** : coordinates may appear in arbribratry order
+    - **soa** : coordinates are fused into the parent level in SoA (structure of array)
```

Comment:
You will have to add some text around L158 that by default, singleton is stored in AoS format, but that this default can be overwritten with the SoA property (similar to how we say that by default, stuff is ordered / unique and we can override that)